src/github-client.js

```diff
@@ -40,6 +40,31 @@ function githubClient(github, context) {
       return data
     },
 
+    async enableAutoMergePullRequest(pullRequestId, mergeMethod) {
+      const query = `
+mutation ($pullRequestId: ID!, $mergeMethod: PullRequestMergeMethod!) {
+  enablePullRequestAutoMerge(input: {
```

Comment:
Where did you tested it?
I wonder if the GHUser that runs this GHA has the permission to execute this mutation

Reply:
Tried in a private repo of the company I work for but I'm gonna create a public repo and paste the url soon.

Reply:
You can see it in action https://github.com/shouichi/github-action-merge-dependabot/pull/1